tests/functional/spec/census/routing/another-address.spec.js

```diff
@@ -156,4 +156,35 @@ describe('Another address routing Scenarios', function () {
     expect(AddressType.isOpen()).to.equal(true, 'Expecting go to address type')
   })
 
+  it('Given I am answering question 5 in the individual detail section -Do you stay at another address for more than 30 days a year?, When I dont select an answer, Then I am routed to 7. Are you a schoolchild or student in full-time education', function () {
+    startCensusQuestionnaire('census_household.json', true)
+    PermanentOrFamilyHome.clickPermanentOrFamilyHomeAnswerYes().submit()
+    HouseholdComposition.setFirstName('John').addPerson().setFirstName('Jane', 1).submit()
+    EveryoneAtAddressConfirmation.clickEveryoneAtAddressConfirmationAnswerYes().submit()
+    OvernightVisitors.setOvernightVisitorsAnswer(0).submit()
+    HouseholdRelationships.clickHouseholdRelationshipsAnswerHusbandOrWife().submit()
+    WhoLivesHereCompleted.submit()
+
+    // household-and-accommodation
+    TypeOfAccommodation.clickTypeOfAccommodationAnswerWholeHouseOrBungalow().submit()
```

Comment:
You can use the helper function to be less verbose here in `complete-section.js`